BUILDING.md

```diff
@@ -43,6 +43,8 @@ need to satisfy this dependencies in your system:
 
 * CentOS/Fedora: `yum install btrfs-progs-devel`
 * Debian/Ubuntu: `apt-get install btrfs-tools`
+	* Debian Buster/Ubuntu 19.10: 
```

Comment:
CI whitespace check failing on the markdown because of the extra space after the colon before the newline.